File: 0125-valid-palindrome/0125-valid-palindrome.c
bool isPalindrome(char* s) {
   int left =0;
   int right= strlen(s)-1;
   while(left<=right){
    if(!isalnum(s[left])){
        left++;
    }
    else if(!isalnum(s[right])){
        right--;
    }
    else{
    if(tolower(s[right])!=tolower(s[left])){
        return false;
    }
    right--;
    left++;
    }

   }
   return true;
}

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        s1=""
        for x in s:
            if x.isalnum():
                s1+=x.lower()
        return s1==s1[::-1]        
